src/python/WMCore/Services/PhEDEx/PhEDEx.py

```diff
@@ -32,7 +36,7 @@ def __init__(self, httpDict=None, responseType="json", logger=None,
         Service.__init__(self, httpDict)
 
     def _getResult(self, callname, clearCache=False,
-                   args=None, verb="POST"):
+                   args='', verb="POST"):
```

Comment:
Since args (if provided) is a list data type, you should not change its data-type here. The None is correct default for that.

Reply:
If you look for the encoded_data downstream (at Requests or pycurl_manager level), you'll see that the default value is an empty string. But switching it back to None won't harm either...